blaseball_mike/models.py

```diff
@@ -418,7 +418,7 @@ def blessing_id(self):
         return self.bonus_id
 
 
-class OffseasonResult(Base):
+class ElectionResult(Base):
```

Comment:
Since this is out in the wild, can you keep the `OffseasonResult` class and create a new `ElectionResult` class that subclasses it to act as an alias? Or we can do a major revision as a breaking change.

Reply:
I'll do a subclass for it and OffseasonSetup